public/tradingview-chart.css

```diff
@@ -37,15 +37,21 @@ html.theme-dark .menuWrap-g78rwseC {
   background-color: #20243f !important;
 }
 
-html.theme-dark .item-2IihgTnv {
+html.theme-dark .item-2IihgTnv:not(.isActive-2IihgTnv) {
   cursor: pointer;
   background-color: #20243f !important;
   color: #b2b5be !important;
 }
-html.theme-dark .item-2IihgTnv:hover {
+
+html.theme-dark .item-2IihgTnv:not(.isActive-2IihgTnv):hover,
+html.theme-dark .container-3Ywm3-oo.selected-3Ywm3-oo {
```

Comment:
hm, looks like generated classNames, probably will change if tv lib will be updated? Just wondering, not sure it can be fixed easily

Reply:
Yes, when the version will be updated it will change. We needed those changes and there is no other way to make that happen. Once tv is stable and will not require changes I will look into this again.